nanoFramework.Json.Test/JsonUnitTests.cs

```diff
@@ -1539,7 +1539,7 @@ public void SerializeCosmosDbObject_02()
         public void SerializeStringWithNewLine_Should_ReturnWithoutNewLine()
         {
             var inputData = "multiline\nstring";
-            var expectedJson = "\"multiline\nstring\"";
+            var expectedJson = "\"multiline\\\nstring\"";
```

Comment:
I'm sorry, but this is not correct - this string still has a line break in it _plus_ an extra backslash coming out of nowhere:
![grafik](https://user-images.githubusercontent.com/21185621/179425606-fac9b0ba-9369-429b-9adc-eaa3e6b7c506.png)
It has to be ``var expectedJson = "\"multiline\\nstring\""`` - only in this version, the "\n" isn't interpreted as a line break by anything but the JSON parser, and only this version would actually be accepted by a JSON parser:
![grafik](https://user-images.githubusercontent.com/21185621/179425723-b3b6d617-1391-40aa-a0b0-82028f283be2.png)